### PR TITLE
providing code examples on `ItemList`'s attributes and basic methods

### PR DESCRIPTION
**Pull request by EmbraceLife**

### CODE REVIEW

File: docs_src/data_block.ipynb [hunk 1]
```diff
@@ -1023,6 +1023,356 @@
     "  - [`CollabList`](/collab.html#CollabList) for collaborative filtering"
```
Comment (sgugger) [inline docs_src/data_block.ipynb]: <p>The command `repr(il_data)` calls `__repr__` under the hood. We don't need to document this here as this is standard python. I would just show the `il.data` object and live it at that.</p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/2026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Comment (sgugger) [inline docs_src/data_block.ipynb]: <p>Same, don't document `__len__` as it's standard python. </p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/2026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Comment (sgugger) [inline docs_src/data_block.ipynb]: <p>Again, `__getitem__` is standard python.</p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/2026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='70'/>

Comment (sgugger) [inline docs_src/data_block.ipynb]: <p>The `get` method is documented later in this notebook I believe, in the section explaining how to create an `ItemList`.</p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/2026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>

Comment (sgugger) [inline docs_src/data_block.ipynb]: <p>I'd show that there are new items inside too.</p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/2026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='78'/>

**Comment by sgugger** [comment]
Thanks for those, but this is the documentation of fastai. You don't need to talk about standard python methods that are well documented elsewhere.

File: docs_src/data_block.ipynb (hunk 1 above)
Comment (EmbraceLife) [reply docs_src/data_block.ipynb]:
```suggestion
    "  - [`CollabList`](/collab.html#CollabList) for collaborative filtering"
```

Comment (EmbraceLife) [reply docs_src/data_block.ipynb]: I see thanks!

Comment (EmbraceLife) [reply docs_src/data_block.ipynb]: I got it.

Comment (EmbraceLife) [reply docs_src/data_block.ipynb]: Thanks, I will work on `get` in a separate code demo.

Comment (EmbraceLife) [reply docs_src/data_block.ipynb]: I don't get this comment, could you elaborate on the changes I need to do? thanks!

Comment (EmbraceLife) [reply docs_src/data_block.ipynb]: I see. 
By the way, I remembered you have figured out a way to show changes in jupyter notebook nicely when working with git. But I don't remember where to find it, could you paste me a link to it? Thanks

Comment (EmbraceLife) [reply docs_src/data_block.ipynb]: Ok, I got this, thanks!

**Pull request closed without merging**